tests/lax_metal_test.py

```diff
@@ -302,13 +302,11 @@ def testCountNonzero(self, shape, dtype, axis):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=False)
     self._CompileAndCheck(jnp_fun, args_maker)
 
-  @jtu.sample_product(shape=all_shapes, dtype=all_dtypes)
+  @jtu.sample_product(shape=nonzerodim_shapes, dtype=all_dtypes)
```

Comment:
OOC why did you need to reduce coverage? Were new failures introduced in jaxlib nightly?

Reply:
Ah. More reason to sync up the test files!